docs/src/components/ComponentDoc/PerfChart/PerfChart.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { FlexibleXYPlot, XAxis, YAxis, HorizontalGridLines, LineSeries, Crosshair } from 'react-vis'
+
+const ChartTooltip = ({ x, data, ...rest }: { x: number; data: any }) => {
```

Comment:
We have a rule in ESLint that disallows to have multiple components in one file

Reply:
Each component is now in its own file.